assets/js/members.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Parse URL parameter
+ *
+ * @param name {string} Parameter name
+ * @param url {string} URL to be parsed
+ * @returns {string|null} Parameter value
+ */
+const getParameterByName = (name, url) => {
+  const urlToParse = url || window.location.href;
+  const nameToParse = name.replace(/[[\]]/g, '\\$&');
+  const regex = new RegExp(`[?&]${nameToParse}(=([^&#]*)|&|#|$)`);
+  const results = regex.exec(urlToParse);
+
+  if (!results) {
+    return null;
+  }
+
+  if (!results[2]) {
+    return '';
+  }
+
+  return decodeURIComponent(results[2].replace(/\+/g, ' '));
+};
+
+const toggleContent = (triggerEl) => {
+  const targetEl = document.querySelector(triggerEl.getAttribute('data-toggle'));
+  const isTargetVisible = !targetEl.hasAttribute('hidden');
+  triggerEl.setAttribute('aria-expanded', !isTargetVisible);
+  targetEl.toggleAttribute('hidden');
+};
+
+// Handle toggle elements.
+const toggleElements = document.querySelectorAll('[data-toggle]');
+
+if (toggleElements.length) {
+  toggleElements.forEach((element) => {
+    element.addEventListener('click', (event) => toggleContent(event.currentTarget));
+  });
+}
+
+// Get URL parameters
+const action = getParameterByName('action');
+const success = getParameterByName('success');
+
+// Display notifications according to URL parameters
+if (action === 'subscribe' && success) {
+  document.querySelector('#notification-subscribe-success').removeAttribute('hidden');
+} else if (action === 'subscribe') {
+  document.querySelector('#notification-subscribe-error').removeAttribute('hidden');
+}
+
+// Handle dismissible elements.
+const dismissibleElements = document.querySelectorAll('[data-dismissible]');
+
+if (dismissibleElements.length) {
+  dismissibleElements.forEach((element) => {
+    element.querySelector('[data-dismiss]').addEventListener('click', (e) => {
```

Comment:
Co když tam ale žádný element s data atributem `data-dismiss` nebude? 🤭  Raději bych to ošetřila 👩🏼‍⚕️